shell/platform/windows/flutter_windows_engine.h

```diff
@@ -0,0 +1,86 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_WINDOWS_FLUTTER_WINDOWS_ENGINE_H_
+#define FLUTTER_SHELL_PLATFORM_WINDOWS_FLUTTER_WINDOWS_ENGINE_H_
+
+#include <memory>
+#include <vector>
+
+#include "flutter/shell/platform/common/cpp/incoming_message_dispatcher.h"
+#include "flutter/shell/platform/windows/flutter_project_bundle.h"
+#include "flutter/shell/platform/windows/public/flutter_windows.h"
+#include "flutter/shell/platform/windows/win32_task_runner.h"
+#include "flutter/shell/platform/windows/window_state.h"
+
+namespace flutter {
+
+class FlutterWindowsView;
+
+// Manages state associated with the underlying FlutterEngine.
```

Comment:
Since there is "Windowsness" here would be good to have a bit more detail in this comment.. Is this class adding "headedness" to FlutterEngine?  Similar to comment on engine, might be good to add a bit more to this comment

Reply:
Expanded slightly. The `Windows` in the name is to indicate its relationship to `flutter_windows` and `FlutterWindowsView`; `flutter_windows` has two major support classes, the view part (`FlutterWindowsView`) and the engine part (`FlutterWindowsEngine`). (And, honestly, because with `FlutterEngine` already being a name in the public API surface, I can't call it that without changing the namespace used internally. I might have to do that eventually, but for now this seemed like a simple way to differentiate them.)